components/checkbox/stories/template.js

```diff
@@ -91,3 +91,70 @@ export const Template = ({
 		</label>
 	`;
 };
+
+/* Shows multiple checkboxes in various states of selected, unselected, indeterminate, etc. */
+export const DocsCheckboxGroup = (args, context) => html`
+	<div style="display: flex; flex-direction: column; padding: 16px">
```

Comment:
I definitely support showing multiple examples in the doc block but I'd love to get rid of the div container here with custom styles because it's harder to maintain consistency if each story uses their own flex container settings. Maybe we can use the `Container` function from the `@spectrum-css/preview/decorators` import? Would need to use:

```
Container({
    direction: "column",
    withBorder: false,
    content: html`
        ${Template...}
        ${Template...}
    `
});
```

Reply:
Oh sure! I had to export it from `utilities.js`, but that's a great idea, as usual! Here's [checkbox](https://github.com/adobe/spectrum-css/pull/2793/commits/06fa4e2e36ad3d83a2855228854ceedd8657a1b0) and [coach indicator.](https://github.com/adobe/spectrum-css/pull/2793/commits/80cee339f891d192f8ec5b79b9b883e4cc39a942)

I also had divs, custom styles & headings in the `AllVariantsCheckboxGroup` (which contained multiple `DocsPageCheckboxGroup`s). I refactored to use `Container()` in that as well- what do you think of those changes? 

<img width="698" alt="Screenshot 2024-08-21 at 3 32 41 PM" src="https://github.com/user-attachments/assets/80b995d1-76e2-431a-a47d-804ae362acdd">

What would you think about me adding: 
```
wrapperStyles: {
    "column-gap": "16px"
},
``` 
to squish the groups together (they have `column-gap: 40` currently), and get them back onto one line at large screens? That's not necessary so it's no big deal if we keep it off for now.

<img width="895" alt="Screenshot 2024-08-21 at 3 33 08 PM" src="https://github.com/user-attachments/assets/a8c58d6f-e2d7-4f78-8584-e7b045a2ee80">